### ISSUE TITLE
Feature request: Slack-style Account Toggling

### ISSUE THREAD

**Issue by electricbrick** [Dimillian/IceCubesApp#1635]
It would be beneficial from a UX, and maybe an accessibility standpoint, to have the ability to toggle between accounts the way one would in apps like Slack or Shift, invoking them via the same shortcuts (CMD + 1, + 2, + 3, and so on).
